package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "vscode-arduino",
```

Comment:
Please merge this repo main branch into yours, because there are some modifications that I hade to do to this file in order to successfully upload the extension to marketplace